typescript-carousel/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TypeScript Carousel</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="row justify-sa align-center">
+      <button id="left-btn" class="direction-btn">
+        <i class="fa-solid fa-circle-chevron-left"></i>
+      </button>
+      <img id="center-img" src="images/001.png" />
+      <button id="right-btn" class="direction-btn">
+        <i class="fa-solid fa-circle-chevron-right"></i>
+      </button>
+    </div>
+    <div id="nav-btn-div" class="row justify-center">
+      <button id="circle-0" class="nav-btn">
+        <i class="fa-solid fa-circle"></i>
+      </button>
+      <button id="circle-1" class="nav-btn">
+        <i class="fa-regular fa-circle"></i>
+      </button>
+      <button id="circle-2" class="nav-btn">
+        <i class="fa-regular fa-circle"></i>
+      </button>
+      <button id="circle-3" class="nav-btn">
+        <i class="fa-regular fa-circle"></i>
+      </button>
+      <button id="circle-4" class="nav-btn">
+        <i class="fa-regular fa-circle"></i>
+      </button>
```

Comment:
It would probably be fine to let these all just be `i`cons and remove the `button`s that are wrapping them.